docs/en/engines/table-engines/index.md

```diff
@@ -79,6 +79,16 @@ Engines in the family:
 - [Buffer](../../engines/table-engines/special/buffer.md#buffer)
 - [KeeperMap](../../engines/table-engines/special/keepermap.md)
 
+### Engine summary
+
+The following comparison table is not meant to be exhaustive.
+
```

Comment:
Suggest to switch x and y axis. There are many more table engines than features.